src/Filters/Filters.jl

```diff
@@ -1,8 +1,9 @@
 module Filters
-import ..DSP: @importffts, filt, filt!
+import ..DSP: @importffts
 using Polynomials, ..Util
 import Base: *
 @importffts
+VERSION >= v"0.7.0-DEV.602" && import ..DSP: filt, filt!
```

Comment:
On 0.6 we need to import these functions from `Base`. Changing this line to 

```julia
if VERSION >= v"0.7.0-DEV.602"
    import ..DSP: filt, filt!
else
    import Base: filt, filt!
end
```

seems to work, but I confess I don't have a full grasp of your FFTW/DSP migration master plan.